### PR TITLE
feat: Allow user to specify a format for stdout T2 (#242)

### PR DESCRIPTION
**Pull request by yogeshtewari**

### CODE REVIEW

File: data_validation/__main__.py [hunk 1]
```diff
@@ -143,6 +143,7 @@ def build_config_managers_from_args(args):
             table_obj,
             labels,
             threshold,
+            format=args.format,
```
Comment (nehanene15) [inline data_validation/__main__.py]: This doesn't need to be an optional parameter. Before we initialize ConfigManager.build_config_manager(), we can set format to args.format similar to the 'threshold' parameter. For example,
```
format = args.format if args.format else 'table'
```

File: tests/unit/result_handlers/test_text.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def test_import(module_under_test):
 def test_basic_result_handler(module_under_test):
     """Test basic handler executes """
     result_df = DataFrame(SAMPLE_RESULT_DATA)
-    result_handler = module_under_test.TextResultHandler()
+    result_handler = module_under_test.TextResultHandler(format="table")
```
Comment (nehanene15) [inline tests/unit/result_handlers/test_text.py]: Can you add a unit test (maybe in unit/test_cli_tools.py) that checks if a user passes in an invalid format an error will be thrown?

**Review by nehanene15 — CHANGES_REQUESTED** [review]

File: tests/unit/result_handlers/test_text.py (hunk 2 above)
Comment (yogeshtewari) [reply tests/unit/result_handlers/test_text.py]: Done.

File: data_validation/__main__.py (hunk 1 above)
Comment (yogeshtewari) [reply data_validation/__main__.py]: Done.

**Review by nehanene15 — APPROVED** [review]

**Pull request merged**